UDP/current_term_courses/v1_student_current_term_course.sql

```diff
@@ -192,5 +197,4 @@ SELECT
  course_section_id,
  canvas_section_id
 FROM
- courses_enrollment_major
-with data
```

Comment:
so dropping the `with data` clause here since Unizin will create those materialized views?

Reply:
We are now doing creating athletic data mart as scheduled query and these are not Materialized views.
https://cloud.google.com/bigquery/docs/scheduling-queries
They might be slight difference but It doesn't seems to be a [big difference](https://cloud.google.com/bigquery/docs/materialized-views-use#:~:text=Scheduled%20queries%20are%20great%20when,reusing%20the%20previously%20computed%20result.) for us. So I just followed, what Unizin suggested.

I have issue open #7 to document scheduled query approach, I will add details in that.